tools/install_prereqs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Build MIGraphX prerequisites for docker container
+
+#install pip3 and rocm-cmake
+sudo apt update && sudo apt install -y python3-pip rocm-cmake
```

Comment:
We should just remove `sudo` here and let the user run the script as sudo instead.

Reply:
changed.